Documentation/ApiOverview/DirectoryStructure/LegacyInstallations.rst

```diff
@@ -17,7 +17,8 @@ see :ref:`Composer-based installations: Directory structure <directory-structure
 Files on project level
 ======================
 
-This folder contains the main entry script :file:`index.php` and might contain
+The TYPO3 project root folder (TYPO3 main) is any subfolder of your web root folder, e.g. :file:`/var/www/html/project1/` or :file:`/home/user1/public_html/project1/`.
```

Comment:
Hmm, what do you mean with "TYPO3 main"? I remember there was a constant some day ago (if I am not wrong).
The sentence can also be misunderstood: "is any subfolder of your web root, e.g. /var/www/html/project1" - At first reading I thought "project1" is the web root (which might not be correct and can be confusing).